src/wdio/services/TerraCommands/validate-element.js

```diff
@@ -0,0 +1,59 @@
+import accessibilityMethods from './accessiblity';
+import visualRegressionMethods from './visual-regression';
+
+/**
+* Helper method to determine the test name name, the element selector, the mismatch tolerance, and the axe rules
+* @param {Array} args - The list of test arguments to parse.
+*/
+const determineOptions = (...args) => {
+  const param1 = args.length ? args[0] : undefined;
+  const param2 = args.length > 1 ? args[1] : undefined;
+
+  let name = 'default';
+  let options = {};
+  if (typeof param1 === 'string') {
+    name = param1;
+    options = typeof param2 === 'object' && !Array.isArray(param2) ? param2 : options;
```

Comment:
i was going to say we could dry this logic up, but if we plan on deprecating the visual regression service this is fine

Reply:
I'd like to deprecate it at some point.